Azrux/Athenas-front
# Add Layout component

## Description
Created Layout component. It includes NavBar and Footer

## Screenshots
![image](https://github.com/Azrux/Athenas-front/assets/89431511/483cbce0-3539-4f64-b2f4-2f23666babc7)

## How to test?
Run the app and go to any page you want. You may see the NavBar and Footer in every page.
